src/library_stack_trace.js

```diff
@@ -5,16 +5,15 @@
  */
 
 var LibraryStackTrace = {
-  $demangle__deps: ['$withStackSave'],
+#if DEMANGLE_SUPPORT
+  $demangle__deps: ['$withStackSave', '__cxa_demangle'],
```

Comment:
```suggestion
  $demangle__deps: ['$withStackSave', '__cxa_demangle', 'free'],
```
May as well also add `free` which I see is used below as well.